handlers/forks.go

```diff
@@ -31,6 +33,21 @@ func Forks(w http.ResponseWriter, r *http.Request) {
 		handlePageError(w, r, pageError)
 		return
 	}
+
+	if r.Header.Get("Content-Type") == "application/json" {
```

Comment:
Should check for the `Accept` header here.
`Content-Type` in requests describes the type of the data being delivered with the request.
Its used for `POST`/`PUT`/`PATCH` requests only

Reply:
Ah gotcha, how does this look to you?